pages/clustering/high-availability.mdx

```diff
@@ -178,6 +179,13 @@ the order isn't important.
 ADD COORDINATOR coordinatorId WITH CONFIG {"bolt_server": boltServer, "coordinator_server": coordinatorServer}; 
 ```
 
+<Callout type="info">
+
+If you are running `ADD COORDINATOR` Cypher queries on coordinator with --coordinator-id=1, you only need to add other two coordinator instances. For example, when using 3 
+coordinators you need to use two `ADD COORDINATOR` queries.
```

Comment:
This is still ambiguous. Can you please give more specific directions, and then also give an example?